haproxy/test/haproxy/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_origin=test
+pkg_name=testhaproxy
+pkg_description="Test plan for the core/haproxy load balancer"
+pkg_version=1.6.11
+pkg_maintainer="The Habitat Maintainers: <humans@habitat.sh>"
+pkg_license=('GPL-2.0' 'LGPL-2.1')
```

Comment:
I suspect any test plans we create should be `Apache-2.0`?
At least, we should not copy the license for the downstream software for our testing code.
Possibly `pkg_license` can just be removed from test plans as they are just testing bits and not shipped components?

Reply:
I think Apache-2.0 is correct, it's what we license the rest of the Habitat projec under.  We're not redistributing the software that we're testing with these packages, and pkg_license doesn't need to include all licenses of pkg_deps and pkg_build_deps.  I might ask Legal on this though.